addons/ace/functions/fnc_setupGlobalArsenal.sqf

```diff
@@ -16,61 +16,98 @@
-			if (EGVAR(radio,personalRadio) in _acreRadios) then {
-				_globalItems pushBackUnique EGVAR(radio,personalRadio);
-			} else {
-				ERROR_1("Setup Global Arsenal radio, invalid radio type: %1",EGVAR(radio,personalRadio));
-			};
+if (GET_CONFIG(acreEnabled,true) && "@ACRE2" call EFUNC(core,checkModPresence)) then {
+	// ACRE radio items
+	private _acreRadios = ([] call acre_api_fnc_getAllRadios) select 0;
+	if (EGVAR(radio,personalRadio) != "NONE") then {
+		if (EGVAR(radio,personalRadio) in _acreRadios) then {
+			_globalItems pushBackUnique EGVAR(radio,personalRadio);
+		} else {
+			ERROR_1("Setup Global Arsenal radio, invalid radio type: %1",EGVAR(radio,personalRadio));
 		};
 	};
+};
 
-	if (GET_CONFIG(haveCTab,true) && "@cTab" call EFUNC(core,checkModPresence)) then {
-	// cTab items
-		_globalItems append ["ItemMicroDAGR", "ItemcTabHCam", "ItemAndroid", "ItemcTab"];
-	};
+if (GET_CONFIG(haveCTab,true) && "@cTab" call EFUNC(core,checkModPresence)) then {
+// cTab items
+	private _defaultList = "['ItemMicroDAGR', 'ItemcTabHCam', 'ItemAndroid', 'ItemcTab']";
+	_globalItems append (parseSimpleArray GET_CONFIG(cTabItems,_defaultList));
+};
 
+if ("@KAT - Advanced Medical" call EFUNC(core,checkModPresence)) then {
 	switch (GET_CONFIG(haveKATMedical,2)) do {
 		// KAT Medical items
 		case 2: {
-			_globalItems append ["kat_aatKit", "kat_accuvac", "kat_guedel", "kat_AED", "kat_X_AED", "kat_larynx", "kat_Pulseoximeter", "kat_chestSeal", "kat_Painkiller", "kat_stretcherBag", "Attachable_Helistretcher", "kat_stethoscope", "KAT_Empty_bloodIV_500", "KAT_Empty_bloodIV_250", "kat_IV_16", "kat_IO_FAST", "kat_amiodarone", "kat_atropine", "kat_lidocaine", "kat_naloxone", "kat_nitroglycerin", "kat_norepinephrine", "kat_phenylephrine", "kat_TXA", "kat_AFAK", "kat_IFAK", "kat_MFAK", "kat_BVM", "kat_pocketBVM", "kat_Carbonate", "kat_oxygenTank_150", "kat_oxygenTank_300", "kat_flumazenil"];
+			private _defaultList = "['kat_aatKit', 'kat_accuvac', 'kat_guedel', 'kat_AED', 'kat_X_AED', 'kat_larynx', 'kat_Pulseoximeter', 'kat_chestSeal', 'kat_Painkiller', 'kat_stretcherBag', 'Attachable_Helistretcher', 'kat_stethoscope', 'KAT_Empty_bloodIV_500', 'KAT_Empty_bloodIV_250', 'kat_IV_16', 'kat_IO_FAST', 'kat_amiodarone', 'kat_atropine', 'kat_lidocaine', 'kat_naloxone', 'kat_nitroglycerin', 'kat_norepinephrine', 'kat_phenylephrine', 'kat_TXA', 'kat_AFAK', 'kat_IFAK', 'kat_MFAK', 'kat_BVM', 'kat_pocketBVM', 'kat_Carbonate', 'kat_oxygenTank_150', 'kat_oxygenTank_300', 'kat_flumazenil']";
+			_globalItems append (parseSimpleArray GET_CONFIG(KATMedicalItems,_defaultList));
 		};
 
 		case 1:
 		{ 
 			_globalItems append ["kat_guedel", "kat_larynx", "kat_chestSeal", "kat_Painkiller", "kat_IV_16", "kat_AED"];
 		};
 	};
+};
+
+private _customItems = (parseSimpleArray GET_CONFIG(customItems,"[]"));
+if (!(_customItems isEqualType [])) then { 
+	["Bad array of custom global arsenal items: %1",_customItems] call BIS_fnc_error;
```

Comment:
Use the ERROR_1 macro for this?  I completely forgot how it works so if it's inappropriate then this is fine too.

Reply:
ERROR only spits to RPT.
Ideally this error shows in game so a mission maker can know their arsenal input is borked

Reply:
using ERROR_MSG_1